src/domains/transaction/components/MultiSignatureRegistrationForm/__snapshots__/MultiSignatureRegistrationForm.test.tsx.snap

```diff
@@ -392,12 +392,12 @@ exports[`MultiSignature Registration Form should render form step 1`] = `
                         data-testid="InputCurrency"
                         name="fee"
                         type="text"
-                        value=""
+                        value="0"
                       />
                     </div>
                   </div>
                   <div
-                    class="absolute bottom-0 px-1 w-full invisible"
+                    class="absolute bottom-0 px-1 w-full"
```

Comment:
@goga-m this snapshot change seems concerning, it was `invisible` before and now it's not. I think it's related to the value above changed from empty to "0". I would review it later when we'll have the sdk updated.

Reply:
@alesmit you are about right :+1: zero fee was not handled and the range slider was visible when the value was 0 in the input